src/fheroes2/system/settings.cpp

```diff
@@ -480,64 +484,64 @@ std::string Settings::String() const
+    os << std::endl << "# First time game run (show additional hints): on/off" << std::endl;
     os << "first time game run = " << ( _gameOptions.Modes( GAME_FIRST_RUN ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# show game intro (splash screen and video): on/off" << std::endl;
+    os << std::endl << "# Show game intro (splash screen and video): on/off" << std::endl;
     os << "show game intro = " << ( _gameOptions.Modes( GAME_SHOW_INTRO ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# enable V-Sync (Vertical Synchronization) for rendering" << std::endl;
+    os << std::endl << "# Enable V-Sync (Vertical Synchronization) for rendering" << std::endl;
     os << "v-sync = " << ( _gameOptions.Modes( GAME_RENDER_VSYNC ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# enable text support mode to output extra information in console window: on/off" << std::endl;
+    os << std::endl << "# Enable text support mode that outputs extra information in console window: on/off" << std::endl;
     os << "text support mode = " << ( _gameOptions.Modes( GAME_TEXT_SUPPORT_MODE ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# enable monochrome (black and white) cursors in the game: on/off" << std::endl;
+    os << std::endl << "# Toggle monochrome (black and white) cursors in the game: on/off" << std::endl;
     os << "monochrome cursor = " << ( _gameOptions.Modes( GAME_MONOCHROME_CURSOR ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# enable 3D audio for objects on Adventure Map: on/off" << std::endl;
+    os << std::endl << "# Enable 3D audio for objects on Adventure Map: on/off" << std::endl;
     os << "3d audio = " << ( _gameOptions.Modes( GAME_3D_AUDIO ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# display system information: on/off" << std::endl;
+    os << std::endl << "# Display system information: on/off" << std::endl;
     os << "system info = " << ( _gameOptions.Modes( GAME_SYSTEM_INFO ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# should auto save be performed at the beginning of the turn instead of the end of the turn: on/off" << std::endl;
+    os << std::endl << "# Perform auto save at the beginning of the turn instead of the end of the turn: on/off" << std::endl;
     os << "auto save at the beginning of the turn = " << ( _gameOptions.Modes( GAME_AUTO_SAVE_AT_BEGINNING_OF_TURN ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# enable cursor software rendering: on/off" << std::endl;
+    os << std::endl << "# Enable cursor software rendering: on/off" << std::endl;
     os << "cursor soft rendering = " << ( _gameOptions.Modes( GAME_CURSOR_SOFT_EMULATION ) ? "on" : "off" ) << std::endl;
 
-    os << std::endl << "# scaling type: nearest or linear (set by default)" << std::endl;
+    os << std::endl << "# Screen scaling type: nearest or linear (default)" << std::endl;
```

Comment:
I suggest to remove this "(default)" part, this information is actually useless for a regular user

```suggestion
    os << std::endl << "# Screen scaling type: nearest or linear" << std::endl;
```